src/mqtt/MQTT.cpp

```diff
@@ -377,6 +381,11 @@ void MQTT::sendSubscriptions()
 #endif // ARCH_NRF52
         }
     }
+#if !MESHTASTIC_EXCLUDE_PKI
+    std::string topic = cryptTopic + "PKI/#";
+    LOG_INFO("Subscribing to %s\n", topic.c_str());
+    pubSub.subscribe(topic.c_str(), 1);
```

Comment:
Let's only subscribe to the "PKI" channel when you have at least one channel downlink enabled.